### PR TITLE
stop bringing in the paper-styles kitchen sink

### PR DESCRIPTION
**Pull request by notwaldorf**
`paper-styles.html` wasn't even used! 🎈

### CODE REVIEW

**Comment by cdata** [comment]
What is this element and why does it have no tests?

File: demo/index.html [hunk 1]
```diff
@@ -17,8 +17,6 @@
     <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
 
     <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
-
-    <link rel="import" href="../../paper-styles/paper-styles.html">
```
Comment (cdata) [inline demo/index.html]: :ski:

**Comment by cdata** [comment]
Minor existential crisis, otherwise LGTM :+1:

**Comment by atotic** [comment]
Re: "What is this element and why does it have no tests?"

This element is a port of core-signals. I consider it an anti-pattern way for elements to forward events. Our only usage used to be in google-signin, and I refactored it out.

My long-term goal is to make this element go away :skull:

**Pull request merged**